datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVAnalyzerRule.scala

```diff
@@ -80,7 +80,7 @@ class MVAnalyzerRule(sparkSession: SparkSession) extends Rule[LogicalPlan] {
   }
 
   def isValidPlan(plan: LogicalPlan, catalog: SummaryDatasetCatalog): Boolean = {
-    !plan.isInstanceOf[Command] && !isDataMapExists(plan, catalog.listAllSchema()) &&
+    !plan.isInstanceOf[Command] && isDataMapExists(plan, catalog.listAllSchema()) &&
```

Comment:
This check is wrong. what exactly the problem you are facing? can you add some test case

Reply:
simply i have created a table in spark-2.2 cluster and after load , updating the record it goes to rewrite the plan. even no datamap created on the same table.